examples/flutter_gallery/android/.gitignore

```diff
@@ -8,6 +8,6 @@
 /captures
 GeneratedPluginRegistrant.java
 
-/gradle
+gradle-wrapper.jar
```

Comment:
I've added a comment to the `.gitignore` files explaining why they are excluded in the examples, and that it is not to be emulated.